### ISSUE TITLE
When misspellings include UTF-8 punctuation, credo chokes on displaying the issue

### ISSUE THREAD

**Issue by chriscrabtree** [Baradoy/credo_spellchecker#3]
It looks like the output below when it happens.

Credo expects the issue `message` to be a normal binary string. When the plugin sends it up a bitstring, it has trouble applying its line-wrapping logic to the bitstring. Thus, the error.

I believe I've fixed this in my fork already, and will push a PR your way once I've tested it in one more scenario that just occurred to me.

```
    (stdlib 5.2) re.erl:806: :re.run(<<70, 111, 117, 110, 100, 32, 109, 105, 115, 115, 112, 101, 108, 108, 101, 100, 32, 119, 111, 114, 100, 32, 96, 103, 97, 114, 114, 121, 226, 96, 46>>, {:re_pattern, 1, 1, 0, <<69, 82, 67, 80, 32, 1, 0, 0, 0, 8, 0, 32, 1, 136, 0, 0, 255, 255, 255, 255, 255, 255, 255, 255, 0, 0, 0, 0, 1, 0, 1, 0, 0, 0, 64, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ...>>}, [{:capture, :all, :binary}, :global, {:offset, 0}])
    (elixir 1.16.0) lib/regex.ex:529: Regex.safe_run/3
    (elixir 1.16.0) lib/regex.ex:516: Regex.scan/3
    (credo 1.7.5) lib/credo/cli/output/ui.ex:60: Credo.CLI.Output.UI.wrap_at/2
    (credo 1.7.5) lib/credo/cli/command/suggest/output/default.ex:209: Credo.CLI.Command.Suggest.Output.Default.do_print_issue/4
    (elixir 1.16.0) lib/enum.ex:987: Enum."-each/2-lists^foreach/1-0-"/2
    (credo 1.7.5) lib/credo/cli/command/suggest/output/default.ex:136: Credo.CLI.Command.Suggest.Output.Default.print_issues_for_category/5
    (elixir 1.16.0) lib/enum.ex:987: Enum."-each/2-lists^foreach/1-0-"/2
```

**Comment by chriscrabtree** [comment]
Scenario test success! PR incoming...

**Comment by Baradoy** [comment]
Closed by #4 

Thanks for the fix @chriscrabtree
